pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -582,8 +582,8 @@ func (m *kubeGenericRuntimeManager) restoreSpecsFromContainerLabels(containerID
 		},
 	}
 	container = &v1.Container{
-		Name:                   l.ContainerName,
-		Ports:                  a.ContainerPorts,
+		Name:  l.ContainerName,
```

Comment:
When I run this locally, `gofmt` doesn't request this change. Also, given that `hack/verify-gofmt.sh` is passing, I think we can be confident this change isn't necessary.